src/data/inchi-dictionary.ts

```diff
@@ -6,89 +6,89 @@ export const INCHIDictionary: Record<string, INCHIProps> = {
     formula: "C2H4O",
     CAS: "75-07-0",
     molecularWeight: 44.05256,
-    inchiString: "",
-    inchiKey: ""
+    inchiString: "nChI=1S/C2H4O/c1-2-3/h2H,1H3",
```

Comment:
```suggestion
    inchiString: "InChI=1S/C2H4O/c1-2-3/h2H,1H3",
```

Reply:
@maneetgoyal sir fixed this issue in second batch commit